be/src/formats/csv/array_converter.cpp

```diff
@@ -32,29 +32,27 @@ Status ArrayConverter::write_quoted_string(OutputStream* os, const Column& colum
 }
 
 bool ArrayConverter::read_string(Column* column, Slice s, const Options& options) const {
-    if (s.size < 2) {
-        return false;
-    }
-    if (s[0] != '[' || s[s.size - 1] != ']') {
+    auto array_reader = create_array_reader(options);
```

Comment:
You could make `array_reader` as member variable, so you don't have to create it every time invoke `read_string`

Reply:
read_string will be invoked only once.

Reply:
Ok, I make a mistake, I will change it.